### DISCUSSION TITLE
It would be nice to create different projects on the same PocketBase instance

### DISCUSSION THREAD

**Discussion by jgb-solutions** [pocketbase/pocketbase#4243] [Q&A]
I think it would be really nice to be able to just create different projects with  their own collections, auth and everything on the same PocketBase instance. Say I have 2 or more projects that need to store some data and not a lot of complexity; it would be really cool to handle that with PocketBase. It could still use the same DB or just different DBs entirely for each project. Using same DB would be done with a collection's prefix key if using one DB. Kinda like WordPress does it by default (though not a lot of people take advantage of this). 

That way one wouldn't have to create different PocketBase instances. You know, like you have the Postgres DB server serving multiple clients. Not sure if anyone already did this, so it would be nice to know what people think.

**Comment by KweeBoss** [comment]
> I think it would be really nice to be able to just create different projects with  their own collections, auth and everything on the same PocketBase instance. Say I have 2 or more projects that need to store some data and not a lot of complexity; it would be really cool to handle that with PocketBase. It could still use the same DB or just different DBs entirely for each project. Using same DB would be done with a collection's prefix key if using one DB. Kinda like WordPress does it by default (though not a lot of people take advantage of this). 
> 
> 
> 
> That way one wouldn't have to create different PocketBase instances. You know, like you have the Postgres DB server serving multiple clients. Not sure if anyone already did this, so it would be nice to know what people think.

I just create collection name with prefix and set another auth  table to control. I am sharing 3 sites in one instance and set nginx proxy to serve them with respective domain in order to get SSR cookie for first party.
